### PR TITLE
Return wallet connection details object instead of connection string

### PR DESCRIPTION
**Pull request by louneskmt**
This PR changes the structure of returned connection details by the manager API.

Needed by https://github.com/getumbrel/umbrel-dashboard/pull/267.

### CODE REVIEW

File: logic/system.js [hunk 1]
```diff
@@ -29,8 +29,8 @@ async function getHiddenServiceUrl() {
 async function getElectrumConnectionDetails() {
     try {
         const address = await diskLogic.readElectrumHiddenService();
-        const port = constants.ELECTRUM_PORT;
-        const connectionString = `${address}:${constants.ELECTRUM_PORT}:t`;
+        const port = String(constants.ELECTRUM_PORT);
```
Comment (lukechilds) [inline logic/system.js:32]: Why do you need to cast to string here?

File: logic/system.js [hunk 2]
```diff
@@ -45,9 +45,11 @@ async function getBitcoinP2PConnectionDetails() {
     try {
         const address = await diskLogic.readBitcoinP2PHiddenService();
         const port = String(constants.BITCOIN_P2P_PORT);
+        const connectionString = `${address}:${port}`;        
         return {
             address,
-            port
+            port,
+            connectionString
```
Comment (lukechilds) [inline logic/system.js:52]: This seems redundant. Why do we need to do this server side?

File: logic/system.js [hunk 3]
```diff
@@ -45,9 +45,11 @@ async function getBitcoinP2PConnectionDetails() {
     try {
         const address = await diskLogic.readBitcoinP2PHiddenService();
         const port = String(constants.BITCOIN_P2P_PORT);
+        const connectionString = `${address}:${port}`;
```
Comment (lukechilds) [inline logic/system.js:48]: This seems redundant. Why do we need to do this server side?

File: logic/system.js (hunk 1 above)
Comment (louneskmt) [reply logic/system.js:32]: Not necessary, but with this all the object elements are strings.

File: logic/system.js (hunk 3 above)
Comment (louneskmt) [reply logic/system.js:48]: I wrote this a little while ago (2 months haha), so I assume it was to keep consistency with the other connection details objects. We can remove this if you think it's better.

**Comment by lukechilds** [comment]
@louneskmt are you able to enable edits from maintainers so I can push to this branch?

**Review by lukechilds — APPROVED** [review]

**Pull request merged**